docs-v2/pages/connected-accounts/oauth-clients.mdx

```diff
@@ -0,0 +1,56 @@
+import Callout from '@/components/Callout'
+import ArcadeEmbed from '@/components/ArcadeEmbed'
+import { Steps } from 'nextra/components'
+
+# OAuth Clients
+By default, most OAuth apps on Pipedream use Pipedream's official OAuth client. That means when you connect an account, you'll see Pipedream's logo and name in the OAuth authorization screen along with a set of requested permissions (scopes).
+- Pipedream apps solve for a broad range of use cases, which means the scopes requested by Pipedream's official OAuth client may request a different set of scopes than your specific use case.
+- To define the exact scope of access you'd like to grant, you can configure a custom OAuth client for most OAuth apps in Pipedream.
+
+## Configuring Custom OAuth Clients
```

Comment:
As a matter of style I only capitalize the first word and proper nouns / words that are traditionally capitalized, so I'd recommend taking a look at the rest of the headers, but here I'd change to:

```markdown
## Configuring custom OAuth clients
```